### PR TITLE
enable yuuki senpai to notice me

### PR DESCRIPTION
**Pull request by thejsa**
shitty code is shitty

### CODE REVIEW

File: modules/extra/joke.rb [hunk 1]
```diff
@@ -26,5 +26,21 @@ module Extra
           event.respond("Confucious say #{File.readlines('text/Jokes/confucious.txt').sample.chomp}")
         }
     )
+
+    $cbot.add_command(:notice,
+        code: proc { |event, args|
+          if args.length < 1 || args[0] == 'me'
+            whom = event.user.name
```
Comment (spotlightishere) [inline modules/extra/joke.rb]: For future code maturity, consider using `whom'st've`.

File: modules/extra/joke.rb [hunk 2]
```diff
@@ -26,5 +26,21 @@ module Extra
...
+          else
+            whom = event.bot.parse_mention(args).name
+          end
+          
+          if args.length >= 2 && args[1] == 'senpai'
+            event.respond("*Senpai notices #{whom}*")
+          else
+            event.respond("*notices #{whom}*")
```
Comment (spotlightishere) [inline modules/extra/joke.rb]: *notices your commit* OwO

File: modules/extra/joke.rb [hunk 3]
```diff
@@ -26,5 +26,21 @@ module Extra
...
+          end
+        }
+    )
```
Comment (spotlightishere) [inline modules/extra/joke.rb]: same actually

File: modules/extra/joke.rb (hunk 1 above)
Comment (Seriell) [reply modules/extra/joke.rb]: no

File: modules/extra/joke.rb (hunk 3 above)
Comment (spotlightishere) [reply modules/extra/joke.rb]: abus@gothob.com !!!

**Pull request merged**